### ISSUE TITLE
fix or remove the package system

### ISSUE THREAD

**Issue by soundmud** [soundmud/soundrts#65]
The downloadable packages system was supposed to ease the publishing of mods and maps, but since it isn't complete it delayed the project and this is counterproductive. It might be necessary to remove it for now.

Maybe a separate launcher-updater project would do the trick someday, but the problem is also that curating packages is an additional job which takes time and requires social/managerial skills.

Just providing the core project and letting contributors release mods and maps elsewhere might be a not-so-bad solution.

**Comment by soundmud** [comment]
The package system have been removed. Note: mods will still be easier to install because the user folder can be opened from the options menu.

**Comment by sanslash332** [comment]
The package system sounds a very good solution to the install of new mods.

 

If you remove it, instead of the package system, can you include in the game a auto-discovery function, that can detect the mods, and  on the game, put a mods menus where you list the detected mods an the players can activate o unactivatethem :P

**Comment by soundmud** [comment]
That's practically what the current development version does.

**Comment by sanslash332** [comment]
A OK :P

 

But, you don’t think that is a good option?

 

For noob users, manage the activated mods from a menu, is easier than modify a text file with a sertain format.

 

:p

**Comment by soundmud** [comment]
Let me rephrase it: I think that this is a good option, and that's why I have already implemented it.

**Comment by sanslash332** [comment]
A! Well! Well! OK OK.

 

I understand that you are thinking to remove any method of manage  mods from the game menu xd. Not only remove the online package system :P

 

 

But now, all are well. Thanks for the aclaration.
